examples/03-javascripts-objects-classes/mars_rover.js

```diff
@@ -14,6 +14,81 @@
 
 class MarsRover {
 
+  constructor(x, y) {
+    this.positionX = x;
+    this.positionY = y;
+    this.direction = 'North';
+  }
+
+  position() {
+    return (
+      {
+        x: this.positionX,
+        y: this.positionY,
+        direction: this.direction
+      }
+    )
+  }
+
+  move() {
+    if (this.direction == 'North') {
+      this.positionX = this.position().x;
+      this.positionY = this.position().y +  1;
+    } else if (this.direction == 'Left') {
+      this.positionX = this.position().x -  1;
+      this.positionY = this.position().y;
+    } else if (this.direction == 'Right') {
+      this.positionX = this.position().x +  1;
+      this.positionY = this.position().y;
+    } 
+    return (
+      {
+        positionX: this.positionX, 
+        positionY: this.positionY,
+        direction: this.direction
+      }
+    );
+  }
+
+  turnLeft() {
+    this.positionX = this.position().x;
+    this.positionY = this.position().y;
+    this.direction = 'Left';
+    return (
+      {
+        positionX: this.positionX, 
+        positionY: this.positionY,
+        direction: this.direction
+      }
+    );
+  }
+
+  turnRight() {
+    this.positionX = this.position().x;
+    this.positionY = this.position().y;
+    this.direction = 'Right';
```

Comment:
@anneilustre I think this always return `Right`. If you `turnRight` twice, this will still return `Right` instead of `South`.